src/module/Entity/src/Entity/Entity/Entity.php

```diff
@@ -253,6 +253,11 @@ public function isUnrevised()
         return $this->countUnrevised() > 0;
     }
 
+    public function isDead()
+    {
+        return !$this->getHead();
```

Comment:
Shouldn't this also return `!$this->getHead() || $this->isTrashed()`? Otherwise I see no real point in adding an alias function.